<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xml_file</class>
 <widget class="QDialog" name="xml_file">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>491</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>911</width>
    <height>531</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Работа с форматом XML</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(240, 243, 250);</string>
  </property>
  <widget class="QPushButton" name="btn_exit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>450</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(138, 174, 224);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>10</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Small</family>
     <pointsize>11</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(213, 222, 239);
color: rgb(57, 88, 134);</string>
   </property>
   <property name="text">
    <string>Информация</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>511</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Small</family>
     <pointsize>11</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(213, 222, 239);
color: rgb(57, 88, 134);</string>
   </property>
   <property name="text">
    <string>4. Работа с форматом XML (модуль xml.etree.ElementTree)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="tE_infa">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>40</y>
     <width>351</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(57, 88, 134);</string>
   </property>
  </widget>
  <widget class="QToolBox" name="toolBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>501</width>
     <height>291</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Small</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(183, 206, 243);
color: rgb(57, 88, 134);
</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="pg_create">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>501</width>
      <height>151</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Создание пустого XML-файла</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sitka Small</family>
       <pointsize>11</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(240, 243, 250);
</string>
     </property>
     <property name="text">
      <string>Путь:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lE_create_filepath">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>0</y>
       <width>361</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(57, 88, 134);</string>
     </property>
    </widget>
    <widget class="QToolButton" name="tB_create_filepath">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>0</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(138, 174, 224);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sitka Small</family>
       <pointsize>11</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(240, 243, 250);</string>
     </property>
     <property name="text">
      <string>Имя файла:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lE_create_name">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>341</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(57, 88, 134);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_create">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>60</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(138, 174, 224);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Создать</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>501</width>
      <height>151</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Добавить данные</string>
    </attribute>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sitka Small</family>
       <pointsize>11</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(240, 243, 250);</string>
     </property>
     <property name="text">
      <string>Текст:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lE_write_text">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>391</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(57, 88, 134);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_write">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>120</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(138, 174, 224);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Записать</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sitka Small</family>
       <pointsize>11</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(240, 243, 250);</string>
     </property>
     <property name="text">
      <string>Файл:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lE_write_filepath">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>361</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(57, 88, 134);</string>
     </property>
    </widget>
    <widget class="QToolButton" name="tB_write_filepath">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>10</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(138, 174, 224);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>481</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sitka Small</family>
       <pointsize>11</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(240, 243, 250);</string>
     </property>
     <property name="text">
      <string>Тег:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lE_write_teg">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>391</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(57, 88, 134);</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>501</width>
      <height>151</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Прочитать файл</string>
    </attribute>
    <widget class="QPushButton" name="btn_read">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>40</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(138, 174, 224);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Читать</string>
     </property>
    </widget>
    <widget class="QToolButton" name="tB_read_filepath">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>10</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(138, 174, 224);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sitka Small</family>
       <pointsize>11</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(240, 243, 250);</string>
     </property>
     <property name="text">
      <string>Файл:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lE_read_filepath">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>361</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(57, 88, 134);</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>501</width>
      <height>151</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Удалить файл</string>
    </attribute>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sitka Small</family>
       <pointsize>11</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(240, 243, 250);</string>
     </property>
     <property name="text">
      <string>Файл:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QToolButton" name="tB_delete_filepath">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>10</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(138, 174, 224);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lE_delete_filepath">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>361</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(57, 88, 134);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_delete">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>40</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(138, 174, 224);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
